src/schema/v2/me/delete_account_mutation.ts

```diff
@@ -0,0 +1,82 @@
+import { GraphQLBoolean, GraphQLNonNull, GraphQLObjectType, GraphQLString, GraphQLUnionType } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { ResolverContext } from "types/graphql"
+
+const AccountMutationDeleteSuccessType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "AccountMutationSuccess",
+  isTypeOf: (data) => {
+    return data._type !== "GravityMutationError"
+  },
+  fields: () => ({
+    success: {
+      type: GraphQLBoolean,
+      resolve: () => true,
+    },
+  }),
+})
+
+const AccountMutationFailureType = new GraphQLObjectType<any, ResolverContext>({
+  name: "AccountMutationFailure",
+  isTypeOf: (data) => {
+    return data._type === "GravityMutationError"
+  },
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: (err) => (typeof err.message === "object" ? err.message : err),
+    },
+  }),
+})
+
+export const AccountMutationType = new GraphQLUnionType({
+  name: "AccountMutationType",
+  types: [AccountMutationDeleteSuccessType, AccountMutationFailureType],
+})
+
+export const deleteUserAccountMutation = mutationWithClientMutationId<
+  any,
+  any,
+  ResolverContext
+>({
+  name: "DeleteAccount",
+  description: "Delete User Artsy Account",
+  inputFields: {
+    explanation: {
+      description: "Reason for deleting the account.",
+      type: new GraphQLNonNull(GraphQLString)
+    },
+    url: {
+      description: "Referrer location",
+      type: new GraphQLNonNull(GraphQLString)
```

Comment:
```suggestion
      type: GraphQLString
```

Reply:
@dzucconi Using only the `GraphQLString` causes us to run into an error of: `This expression is not constructable. Type 'GraphQLScalarType' has no construct signatures.`

Reply:
That shouldn't be the case but I'm AFK until later. There should be plenty of examples in the codebase if you take a look around.

Reply:
Changed.